README.md

```diff
@@ -1 +1,13 @@
 # dmpro2014.github.io
+
+## Setup
+
+**Prerequisites:** ruby, rubygems and bundler.
+
+When these are installed, you can run `bundle install`.
```

Comment:
Jekyll 2 requires ruby 1.9.3 or higher, according to http://jekyllrb.com/docs/installation/